wagtail/admin/templates/wagtailadmin/pages/workflow_history/detail.html

```diff
@@ -29,132 +29,136 @@ <h2>{% icon "clipboard-list" class_name="initial" %} {{ workflow_state.workflow.
-                                        {% endblocktrans %}
-                                    {% endif %}
-                                {% endif %}
-                            {% endif %}
-                        </td>
-                        {% for task_state in task_states %}
-                            <td>
-                                {% if task_state.status == 'approved' or task_state.status == 'rejected' %}
-                                    {% if task_state.finished_by %}
-                                        {% blocktrans trimmed with action='<div class="status-tag primary">'|add:task_state.get_status_display|add:'</div>'|safe who=task_state.finished_by|user_display_name at=task_state.finished_at %}
-                                            {{ action }} by <b>{{ who }}</b> at <b>{{ at }}</b>
-                                        {% endblocktrans %}
-                                    {% else %}
-                                        {% blocktrans trimmed with action='<div class="status-tag primary">'|add:task_state.get_status_display|add:'</div>'|safe at=task_state.finished_at %}
-                                            {{ action }} at <b>{{ at }}</b>
-                                        {% endblocktrans %}
-                                    {% endif %}
-                                {% else %}
-                                    <div class="status-tag primary">{{ task_state.get_status_display }}</div>
-                                {% endif %}
-                                {% if task_state.specific.get_comment %}
-                                    <p>{% trans 'with comment:' %} <b>"{{ task_state.specific.get_comment }}"</b></p>
-                                {% endif %}
-                            </td>
+        <div class="w-tabs" data-tabs>
+            <div class="w-tabs__wrapper">
+                <div role="tablist" class="w-tabs__list nice-padding ">
+                    <a id="tab-label-tasks" href="#tab-tasks" class="w-tabs__tab " role="tab" aria-selected="false" aria-controls="tab-tasks" tabindex="-1" data-tab-button="tasks">
+                        <div data-tabs-errors="" class="w-tabs__errors ">
+                            <span class="w-sr-only">Errors Count: </span>
+                            <span data-tabs-errors-count=""></span>
+                        </div>
+                        {% trans "Tasks" %}
+                    </a>
+                    <a id="tab-label-timeline" href="#tab-timeline"  class="w-tabs__tab " role="tab" aria-selected="true" aria-controls="tab-table" tabindex="-1" data-tab-button="timeline">
+                        <div data-tabs-errors="" class="w-tabs__errors ">
+                            <span class="w-sr-only">Errors Count: </span>
+                            <span data-tabs-errors-count=""></span>
+                        </div>
+                        {% trans "Timeline" %}
+                    </a>
```

Comment:
You should not have to write out this whole link, instead use the include `wagtail/admin/templates/wagtailadmin/shared/tabs/tab_nav_link.html`

Django includes are a way to include another template with some data.

Example
```
{% include 'wagtailadmin/shared/tabs/tab_nav_link.html' with tab_id=child.heading title=child.heading classes=child.classes|join:" " %}
```

```suggestion
                    {% include 'wagtailadmin/shared/tabs/tab_nav_link.html' with tab_id="tab-label-timeline" title=timeline_title %}
```